lang/system/pl.xml

```diff
@@ -314,7 +562,7 @@ System Vita3K jest gotowy!</completed_setup>
 		</date_time>
 		<language name="Język">
 			<system_language>Język systemu</system_language>
-			<input_language name="język wprowadzania">
+			<input_language name="Język wprowadzania">
```

Comment:
You're right.

```suggestion
			<input_language name="Języki wprowadzania">
```

Reply:
Well was gathering Settings names from PS Vita User Guide site sooo